client/components/cancel/stages/executeCancellation.tsx

```diff
@@ -106,11 +106,19 @@ const getCancellationSummaryWithReturnButton = (body: ReactNode) => () =>
 	);
 
 const getCaseUpdatingCancellationSummary =
-	(caseId: string | '', productType: ProductTypeWithCancellationFlow) =>
+	(
+		caseId: string | '',
+		productType: ProductTypeWithCancellationFlow,
+		productFriendlyName: string,
+	) =>
 	(productDetails: ProductDetail[]) => {
 		const productDetail = productDetails[0] || { subscription: {} };
+		console.log('executeCancellation productDetail', productDetail);
```

Comment:
Looks like a sneaky console log found it's way in here

Reply:
Oops! Good spot Rich